### PR TITLE
updated image and updated commit hash for kyverno

### PR DESCRIPTION
**Pull request by pxp928**
Jim updated his image to include the latest kyverno updates. I have pinned the commit to Dec 10th and verified that kyverno is now running properly. Addresses issue #102

### CODE REVIEW

File: platform/30-kyverno-setup.sh [hunk 1]
```diff
@@ -50,7 +50,7 @@ kubectl create secret generic regcred --type=kubernetes.io/dockerconfigjson --fr
 ### failed calling webhook "mutate.kyverno.svc-fail" error. Once this image
 ### is offically released, this will be removed.
 echo -e "${C_GREEN}Patching Kyverno image until release of v1.6.0...${C_RESET_ALL}"
-kubectl set image -n kyverno deploy/kyverno kyverno=ghcr.io/jimbugwadia/kyverno:v1.6.0-130-g8a0d465d
+kubectl set image -n kyverno deploy/kyverno kyverno=ghcr.io/jimbugwadia/kyverno:v1.6.0-latest
```
Comment (bradbeck) [inline platform/30-kyverno-setup.sh:53]: Would something like the following work to pin the version of this image?
```
kubectl set image -n kyverno deploy/kyverno kyverno=ghcr.io/jimbugwadia/kyverno@sha256:c9f1655e61dfc043e37d63c960b762c118485465b4c3def0dacb2d13cf7e4dca
```

Comment (pxp928) [reply platform/30-kyverno-setup.sh:53]: Yes we can do that. As Jim does not update his own image regularly maybe its not needed? @mlieberman85 was saying Kyverno full release was coming in January. What does the team think?

**Comment by bradbeck** [comment]
I'm seeing the following recurring message when looking at `kubectl logs -l 'app=kyverno' -n kyverno -f`
```
I1213 19:31:49.489049       1 status.go:75] WebhookMonitor/WebhookStatusControl "msg"="updating deployment annotation" "name"="kyverno" "namespace"="kyverno" "key"="kyverno.io/webhookActive" "val"="true"
E1213 19:31:49.504765       1 monitor.go:157] WebhookMonitor/webhookMonitor "msg"="failed to annotate deployment webhook status to success" "error"="key kyverno.io/webhookActive, val true: deployments.apps \"kyverno\" is forbidden: User \"system:serviceaccount:kyverno:kyverno-service-account\" cannot update resource \"deployments\" in API group \"apps\" in the namespace \"kyverno\""  
```

**Comment by pxp928** [comment]
> 

Asking Jim if that is an issue with the service account permissions. Ill update if needed.

**Comment by pxp928** [comment]
> Asking Jim if that is an issue with the service account permissions. Ill update if needed.

Updated PR with the new commit. Fixes the issue with service account permissions.

**Review by bradbeck — APPROVED** [review]

**Review by rgreinho — APPROVED** [review]
👍

**Pull request merged**